### ISSUE TITLE
:Tutor shows check mark on to-delete lines

### ISSUE THREAD

**Issue by SimonDold** [neovim/neovim#29514]
### Problem

in :Tutor the check mark is used at lines of the poem that should not be there.

### Steps to reproduce

:Tutor
308j
there we see

```
✅1)  Roses are red,
✅2)  Mud is fun,
✅3)  Violets are blue,
✅4)  I have a car,
✅5)  Clocks tell time,
✅6)  Sugar is sweet
✅7)  And so are you.
```

### Expected behavior

```
✅1)  Roses are red,
❌2)  Mud is fun,
✅3)  Violets are blue,
❌4)  I have a car,
❌5)  Clocks tell time,
✅6)  Sugar is sweet
✅7)  And so are you.
```

### Neovim version (nvim -v)

NVIM v0.10.0

### Vim (not Nvim) behaves the same?

no

### Operating system/version

Ubuntu 22.04.4 LTS

### Terminal name/version

gnome-terminal-

### $TERM environment variable

xterm-256color

### Installation

snap

**Comment by rogererens** [comment]
Also, I'd like to suggest to adapt the third instruction above the poem:
'3. Now move to the fourth line.'
->
'3. Now move to the third (previously fourth) line.'

**Comment by SimonDold** [comment]
I agree that the wording there is not ideal. However, I think this is a separate discussion as the one requires change in the files responsible for the check marks while the other is just about the wording in lines that are not considered by the check marks logic.

**Comment by eduardorittner** [comment]
Apparently tutor's handling of checkmarks on every line is very simple, it has 2 modes: either it's always toggled on or it only toggles with *one* specific string, as far as I can tell it has no support for regular expressions or anything like that. That being said the best I could come up with was this:
![image](https://github.com/user-attachments/assets/c676a088-59f7-4a03-a073-2d0531c1adc7)
And after the wrong lines are deleted:
![image](https://github.com/user-attachments/assets/a2e2c95c-a128-463b-a116-efe88e41140b)
While it's not exactly ideal I think it's still better than the way it currently is.
